workflow-templates/build-test-dotnet-template.properties.json

```diff
@@ -0,0 +1,12 @@
+{
+    "name": "Build and Test .NET",
+    "description": "Workflow that builds and tests your .NET project",
+    "iconName": "octicon gear",
+    "categories": [
+        "continuous-integration",
+        "testing"
+    ],
+    "filePatterns": [
+        "^.*\\.csproj$"
```

Comment:
Does this require a `.csproj` file in the root of the repo? We're trying to standardize on a structure that looks more like:

```
MyProject.sln
+- src/
   +- ModuleA/
   |  +- ModuleA.csproj
   +- ModuleB/
      +- ModuleB.csproj
```
So the proper file to look for would probably be a `*.sln` in the root of the repo.

Reply:
I believe this will just look for any file within the repo that ends with .csproj but I think having it look for the .Sln makes more sense anyway since the workflow requires a sln to be passed into it so I will update it to that anyway

Reply:
The docs say:

> `filePatterns` - Optional. Allows the workflow to be used if the user's repository has a file in its root directory that matches a defined regular expression.

So I guess it depends on how the pattern is interpreted. It looks like to me like "any characters followed by a `.csproj`" so hopefully that also covers directory separators. We should verify how it works, exactly.